cirq/google/xmon_gates.py

```diff
@@ -542,13 +527,11 @@ def inverse(self):
             raise ValueError("Don't have a known inverse.")
         return ExpZGate(half_turns=-self.half_turns)
 
-    def has_matrix(self):
-        return not isinstance(self.half_turns, value.Symbol)
-
-    def matrix(self):
-        if not self.has_matrix():
-            raise ValueError("Don't have a known matrix.")
-        return np.diag([(-1j)**self.half_turns, 1j**self.half_turns])
+    def _unitary_(self) -> Optional[np.ndarray]:
+        if isinstance(self.half_turns, value.Symbol):
+            return None
```

Comment:
Why `None` here instead of `NotImplemented` like the methods above?

Reply:
Good catch. Done.